lib/internal/repl.js

```diff
@@ -61,6 +62,7 @@ function createRepl(env, opts, cb) {
   }
 
   const repl = REPL.start(opts);
+  repl[kIsInternal] = true;
```

Comment:
I think marking objects as “special” in a certain way is an anti-pattern. Is there a reason not to use a real option?

Reply:
The latest version uses the options only and since all functions that use this information are declared inside of the constructor, it's indeed cleaner :+1:.